CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [v2.0.0](http://github.com/ndustrialio/contxt-sdk-js/tree/v2.0.0) (2019-07-17)
+
+**Changed**
+
+- Change method name
+  - Change .subscribeEvent to .subscribeUser and .unsubscribeEvent to .unsubscribeUser
```

Comment:
Can you change this to be more like most of the other comments, like:

```
**Breaking Change**

- Renamed `Events#subscribeEvent` to `Events#subscribeUser`
- Renamed `Events#unsubscribeEvent` to `Events#unsubscribeUser`

Reply:
👍